hack/kubearchive-install.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Install KubeArchive by itself. Used by quick install and also to update existing
+# KubeArchive installations.
+
+set -o errexit
+set -o xtrace
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd ${SCRIPT_DIR}/..
+
+PODS=$(kubectl -n kubearchive get pods | grep -E -v "NAME|No resources|apiserversource" |& awk '{print $1}')
+
+YAML=$(mktemp --suffix=.yaml -t kubearchive-XXX)
+bash cmd/operator/generate.sh
+helm template kubearchive charts/kubearchive -n kubearchive \
+    --include-crds \
+    --set "global.production=true" > /tmp/kubearchive-not-resolved.yaml
+ko resolve -f /tmp/kubearchive-not-resolved.yaml --base-import-paths > ${YAML}
+kubectl apply -n kubearchive -f ${YAML}
+
+rm -f ${YAML}
+
+kubectl -n kubearchive rollout status deployment --timeout=90s
```

Comment:
I think this would be a good place to echo what is happening in the script, before the waits start.

```suggestion
echo "Start a rollout of the KubeArchive deployment."
kubectl -n kubearchive rollout status deployment --timeout=90s
```

Reply:
This information is in the output, first because the command is echoed and second because `kubectl` prints a status.

```
+ kubectl -n kubearchive rollout status deployment --timeout=90s
Waiting for deployment "kubearchive-api-server" rollout to finish: 0 of 1 updated replicas are available...
deployment "kubearchive-api-server" successfully rolled out
Waiting for deployment "kubearchive-operator" rollout to finish: 0 of 1 updated replicas are available...
deployment "kubearchive-operator" successfully rolled out
deployment "kubearchive-sink" successfully rolled out 
```

Reply:
OKAY